src/sections/Community/Handbook/recognition.js

```diff
@@ -104,96 +108,96 @@ const RecognitionPage = () => {
             </p>
             <ul style={badgeListStyle}>
               <p><b>Activity badges:</b></p>
-              <li>
+              <li style = {badgeSpace}>
```

Comment:
@bunnyDasari isn't it seems very repetitive . There could any other way to style it instead passing repetitive styles each time.

Reply:
I have tried using styled components I think their maybe other way